libnavigation-ui/src/main/java/com/mapbox/navigation/ui/map/NavigationMapboxMap.java

```diff
@@ -258,9 +261,41 @@ public void addDestinationMarker(Point position) {
    * will clear all destination / custom markers that have been added to the map.
    *
    * @param options for the custom {@link com.mapbox.mapboxsdk.plugins.annotation.Symbol}
+   *
+   * @return the {@link com.mapbox.mapboxsdk.plugins.annotation.Symbol} that added to the map or null if adding fails
+   */
+  public Symbol addCustomMarker(SymbolOptions options) {
```

Comment:
This is breaking SemVer. Is that intentional? If so, should we do the same with `addDestinationMarker` for consistency?

Reply:
hmmm, yes, it's intentional. Why we need the symbol instance is in the new Feedback UX flow, if the user doesn't submit a Feedback, we should be able to remove the marker that added when clicking FeedbackButton.

Reply:
What about 👇 
> If so, should we do the same with `addDestinationMarker` for consistency?

Reply:
yeah, I am thinking this just now.
since we are managing it in `NavigationSymbolManager`, I think it might not be a good idea to expose this destination symbol. we may offer an API to clear this if it makes sense?